<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mealdisplay</class>
 <widget class="QMainWindow" name="mealdisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>731</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item alignment="Qt::AlignHCenter">
      <widget class="QRadioButton" name="BreakFastRadio">
       <property name="text">
        <string>BreakFast</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QRadioButton" name="LunchRadio">
       <property name="text">
        <string>Lunch</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QRadioButton" name="DinnerRadio">
       <property name="text">
        <string>Dinner</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QRadioButton" name="LowCarbRadio">
       <property name="text">
        <string>LowCarb</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="HighProtienRadio">
       <property name="text">
        <string>HighProtien</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>960</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="LabelContainer">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>20</number>
       </property>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>View </string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>View</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>View</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>View</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>View</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>390</y>
      <width>661</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>300</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1002</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDiet">
    <property name="title">
     <string>Diet</string>
    </property>
    <addaction name="actionNone"/>
    <addaction name="actionVegan"/>
    <addaction name="actionPescatarian"/>
    <addaction name="actionDairyFree"/>
    <addaction name="actionGlutan"/>
   </widget>
   <addaction name="menuDiet"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNone">
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionVegan">
   <property name="text">
    <string>Vegan</string>
   </property>
  </action>
  <action name="actionPescatarian">
   <property name="text">
    <string>Pescatarian</string>
   </property>
  </action>
  <action name="actionDairyFree">
   <property name="text">
    <string>DairyFree</string>
   </property>
  </action>
  <action name="actionGlutan">
   <property name="text">
    <string>Glutan</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
